<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BanqueWidget</class>
 <widget class="QWidget" name="BanqueWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BanqueWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_informationsBanques">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>611</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations de la banque</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les informations de tous les comptes</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetInfoCompte">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Numéro de compte</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Solde en euros</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ville</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nom de l'agence</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_ConnexionBDD">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>611</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion à la BDD</string>
   </property>
   <widget class="QPushButton" name="pushButton_Connexion">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Connexion à 
la base de données</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>321</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Adresse IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nom de la base de données:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Login:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Mot de passe:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_IP">
         <property name="text">
          <string>172.18.58.8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_BDD">
         <property name="text">
          <string>banque</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Login">
         <property name="text">
          <string>snir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_Mdp">
         <property name="text">
          <string>snir</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
